packages/osmojs/src/codegen/capability/v1/capability.ts

```diff
@@ -95,259 +86,203 @@ export interface CapabilityOwnersAminoMsg {
  * CapabilityOwners defines a set of owners of a single Capability. The set of
  * owners must be unique.
  */
-
 export interface CapabilityOwnersSDKType {
   owners: OwnerSDKType[];
 }
-
 function createBaseCapability(): Capability {
   return {
-    index: Long.UZERO
+    index: BigInt("0")
   };
 }
-
 export const Capability = {
   typeUrl: "/capability.v1.Capability",
-
-  encode(message: Capability, writer: _m0.Writer = _m0.Writer.create()): _m0.Writer {
-    if (!message.index.isZero()) {
+  encode(message: Capability, writer: BinaryWriter = BinaryWriter.create()): BinaryWriter {
+    if (message.index !== BigInt(0)) {
       writer.uint32(8).uint64(message.index);
     }
-
     return writer;
   },
-
-  decode(input: _m0.Reader | Uint8Array, length?: number): Capability {
-    const reader = input instanceof _m0.Reader ? input : new _m0.Reader(input);
+  decode(input: BinaryReader | Uint8Array, length?: number): Capability {
+    const reader = input instanceof BinaryReader ? input : new BinaryReader(input);
     let end = length === undefined ? reader.len : reader.pos + length;
     const message = createBaseCapability();
-
     while (reader.pos < end) {
       const tag = reader.uint32();
-
       switch (tag >>> 3) {
         case 1:
-          message.index = (reader.uint64() as Long);
+          message.index = BigInt(reader.uint64().toString());
```

Comment:
Is thisString conversion needed here for the copy? What does `.uint64()` return?

Reply:
Good catch! I'll fix this right now! Really appreciate!